nav2_dwb_controller/dwb_core/include/dwb_core/dwb_local_planner.hpp

```diff
@@ -55,14 +57,17 @@ namespace dwb_core
  * @class DWBLocalPlanner
  * @brief Plugin-based flexible local_planner
  */
-class DWBLocalPlanner : public nav2_util::LifecycleHelperInterface
+class DWBLocalPlanner : public nav2_core::LocalPlanner
 {
 public:
   /**
    * @brief Constructor that brings up pluginlib loaders
    */
-  DWBLocalPlanner(
-    nav2_util::LifecycleNode::SharedPtr node, TFBufferPtr tf, CostmapROSPtr costmap_ros);
+  DWBLocalPlanner();
+
+  void initialize(
+    nav2_util::LifecycleNode::SharedPtr node,
```

Comment:
Do we require a LifecycleNode? Can this be done with a regular node too? Maybe we add a 2nd initialize function for supporting regular nodes instead?